roles/telemetry_logging/tasks/main.yml

```
@@ -0,0 +1,500 @@
+    nspace: openshift-logging
+    services_list:
+      - cluster-logging-operator-metrics
+      - logging-loki-compactor-grpc
+      - logging-loki-compactor-http
+      - logging-loki-distributor-grpc
+      - logging-loki-distributor-http
+      - logging-loki-gateway-http
+      - logging-loki-gossip-ring
+      - logging-loki-index-gateway-grpc
+      - logging-loki-index-gateway-http
+      - logging-loki-ingester-grpc
+      - logging-loki-ingester-http
+      - logging-loki-querier-grpc
+      - logging-loki-querier-http
+      - logging-loki-query-frontend-grpc
+      - logging-loki-query-frontend-http
+      - logging-view-plugin
+      - openstack-logging
+
+  tasks:
+    - name: Remove "{{ services_out_file }}"
+      ansible.builtin.file:
+        path: "{{ services_out_file }}"
+        state: absent
+      changed_when: false
+
+    - name: Run Services Test in "{{ nspace }}"
+      ansible.builtin.include_tasks: logging_services_tests.yml
+      loop: "{{ services_list }}"
+
+
+- name: Verify openstack services running
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    services_out_file: verify_openstack_services_running_lresults.log
+    nspace: openstack
+    services_list:
+      - nova-internal
+      - nova-metadata-internal
+      - nova-novncproxy-cell1-public
+      - nova-public
+
+  tasks:
+    - name: Remove "{{ services_out_file }}"
+      ansible.builtin.file:
+        path: "{{ services_out_file }}"
+        state: absent
+      changed_when: false 
+
+    - name: Run Services test in "{{ nspace }}" 
+      ansible.builtin.include_tasks: logging_services_tests.yml
+      loop: "{{ services_list }}"
+
+
+- name: Verify packagemanifests exist
+  hosts: localhost 
+  connection: local
+  gather_facts: no
+  vars:
+    manifest_out_file: verify_packagemanifests_exist_lresults.log
+    packagemanifests_list:
+      - "loki-operator 2"
+      - "loki-helm-operator 1"
+
+  tasks:
+    - name: Remove "{{ manifest_out_file }}"
+      ansible.builtin.file:
+        path: "{{ manifest_out_file }}"
+        state: absent
+      changed_when: false
+
+    - name: Run packagemanifest tests
+      ansible.builtin.include_tasks: logging_manifest_tests.yml
+      loop: "{{ packagemanifests_list }}"
+
+
+- name: Verify openshift-operators-redhat subscriptions exist
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    nspace: openshift-operators-redhat 
+    subscript_out_file: verify_openshift-operators-redhat_subscriptions_lresults.log 
+    subscription_list:
+      - loki-operator
+
+  tasks:
+    - name: Remove "{{ subscript_out_file }}"
+      ansible.builtin.file:
+        path: "{{ subscript_out_file }}"
+        state: absent
+      changed_when: false
+
+    - name: Run packagemanifest tests
+      ansible.builtin.include_tasks: logging_subscription_tests.yml
+      loop: "{{ subscription_list }}"
+
+
+- name: Verify ctlplane node containers
+  hosts: ctlplane_nodes
```

EDPM nodes are in the `computes` group, with individual hosts called `compute-0`, `compute-1`, etc

Zuul compiles an inventory file and logs it for convenience:
https://logserver.rdoproject.org/99/99/d89a0b5d52bc318c9ff9d350160d251647101f33/github-check/functional-logging-tests-osp18/37131c1/zuul-info/inventory.yaml

```suggestion
  hosts: ctlplane_nodes
```
```suggestion
  hosts: ctlplane_nodes
```
```suggestion
  hosts: computes
```